js_modules/dagster-ui/packages/ui-core/src/settings/SettingsLeftPane.tsx

```diff
@@ -68,7 +68,7 @@ export const SettingsLeftPane = () => {
           <UserSettingsDialog
             isOpen={showUserSettings}
             onClose={() => setShowUserSettings(false)}
-            visibleFlags={getVisibleFeatureFlagRows()}
+            visibleFlags={useVisibleFeatureFlagRows()}
```

Comment:
This seems like a pretty dangerous place to invoke a hook, since it'd be easy for someone to come and make this portion of the render function conditional or add an early return above 52

Maybe assign this to a local variable so it looks more like a normal hook?

Reply:
You'd get an error if it were called conditionally so I think the risk is fairly non-existant.